store/manifest.go

```diff
@@ -0,0 +1,27 @@
+package store
+
+import (
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+// OpenManifest either initializes a new leveldb database at the path of the string provided
+// or opens an existing leveldb database found at that path.
+func OpenManifest(path string) (*leveldb.DB, error) {
+	db, err := leveldb.OpenFile(path, nil)
+	if err != nil {
+		panic("Could not open manifest")
+	}
+
+	return db, err
```

Comment:
Perhaps we could rename this to `MustOpen`?  `Must` indicates that the function will do what it says or panic. In this case, the function will never return an error, so we could use `MustOpen(path string) *leveldb.DB`.